services/website/website.go

```diff
@@ -84,6 +89,7 @@ func NewWebserver(opts *WebserverOpts) (*Webserver, error) {
 		HeadSlot:                    "",
 		NumPayloadsDelivered:        "",
 		Payloads:                    []*database.DeliveredPayloadEntry{},
+		RelayHash:                   "",
```

Comment:
couldn't you set the hash here once at startup, and not need the `api.cachedRelayHash` at all, or is there any value in recomputing the hash on every update?

Reply:
Yeah, lemme see what I can do. I included it as a method in the webserver interface because I wanted access to the `srv.log` field. That's probably unnecessary. I agree, it would be nicer if it were computed once at startup.

Reply:
Alright yeah, I don't know what I was thinking. Sorry about that.